### ISSUE TITLE
[Bug] [a11y] Talkback does not read the three-dot menu

### ISSUE THREAD

**Issue by softvision-miralobontiu** [mozilla-mobile/fenix#3585]
## Steps to reproduce
1. Turn Talkback on.
1. Tap on three-dot menu.
1. Listen to the description of the menu.

### Expected behavior
The three-dot menu should be described entirely.

### Actual behavior
"Showing items 1 to 4 of 4" without reading/describing the items.

### Device information
* Android device: Samsung Galaxy Tab S3 (Android 8)
* Fenix version: 1.0.0-rc.2 (#11700147)

**Comment by boek** [comment]
This is caused by the separator being a menu item. The idea for fixing it is to try to override the default content description of the menu and hard code the correct amount of items.

**Comment by softvision-miralobontiu** [comment]
The behavior changed, the main three-dot menu's items are read until reaching "Library", then the talkback reads every letter of the "Library", then moves to the next item.
More precisely, it reads:
- What's new,
- Library,
- capital L,
- i India,
- b bravo etc.
After finishing all the letter in "Library", moves to Bookmarks, History etc.
Reproducible on the latest Nightly build from 3/25, and on Beta 4.2.0-beta.2. Tested with OnePlus 5T (Android 9).

**Comment by eliserichards** [comment]
closing as duplicate of #2146
